sections/discussion/ourprocessreflections.tex

```diff
@@ -54,6 +54,20 @@ \subsubsection{Formal reviews}
     \label{fig:formal-reviews}
 \end{figure}
 
+\subsubsection{Prototypes}
+In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding about how the design should be.
+Some of these prototypes has been described in \autoref{sec:prototypes-design}.
+\\\\
+When the development of the prototypes started we had a two people assigned the task of creating hi-fi prototypes, and afterwards people reviewed the prototypes and gave feedback and asked for changes.
+Since the prototypes were high fidelity it was more difficult for reviewers to ask for big changes since it would take a lot of time to change.
+This resulted in there never really being an overall discussion of how the design should be.
```

Comment:
Ved ikke om jeg synes der ikke var en overall discussion, men ved ikke lige hvordan det ellers skal formulers. Der var vel en form for initial discussion, men det gik ikke så meget videre der fra???? 🤔